### PR TITLE
explore cards block

### PR DESCRIPTION
**Pull request by arjun87743**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://explore-cards--world-bank--aemsites.hlx.live/

### CODE REVIEW

File: component-definition.json [hunk 1]
```diff
@@ -150,6 +150,37 @@
...
+            }
+          }
+        },
+        {
+          "title": "Explore Card",
+          "id": "explore-card",
+          "plugins": {
+            "xwalk": {
+              "page": {
+                "resourceType": "core/franklin/components/block/v1/block",
```
Comment (thomsebastin) [inline component-definition.json:176]: Any reason why this is not a block item but block? Nesting blocks is not recommended!!

File: component-models.json [hunk 2]
```diff
@@ -233,5 +233,78 @@
...
+  },
+  {
+    "id": "explore-cards",
+    "fields": [
+      {
+        "component": "richtext",
+        "valueType": "string",
+        "name": "title",
+        "value": "",
+        "label": "Section Title"
```
Comment (thomsebastin) [inline component-models.json:245]: This is a block, so naming it "section title" seems a little misleading.

File: component-models.json [hunk 3]
```diff
@@ -233,5 +233,78 @@
...
+      },
+      {
+        "component": "reference",
+        "valueType": "string",
+        "name": "fileReference",
+        "label": "Background Image",
```
Comment (thomsebastin) [inline component-models.json:251]: All images should have a corresponding alt label. Does the dialogue provide a field for that by default here?

**Review by thomsebastin — CHANGES_REQUESTED** [review]
Looks good. Just left a few comments.

File: .eslintrc.js [hunk 4]
```diff
@@ -21,7 +21,7 @@ module.exports = {
     'xwalk/max-cells': [ // DOC: https://github.com/adobe-rnd/eslint-plugin-xwalk/blob/main/docs/rules/max-cells.md
       'error',
       {
-        '*': 4,
+        '*': 5,
```
Comment (thomsebastin) [inline .eslintrc.js:24]: If you're changing the default to 5, then you can remove the exception added below for hero.

File: component-models.json (hunk 3 above)
Comment (arjun87743) [reply component-models.json:308]: added alt text filed

**Review by thomsebastin — APPROVED** [review]

**Pull request merged**